src/launchpad/analyzers/apple.py

```diff
@@ -66,7 +81,11 @@ def analyze(self, artifact: AppleArtifact) -> AppleAnalysisResults:
         analysis_start_time = time.time()
 
         # Extract basic app information
-        app_info = self._extract_app_info(artifact)
+        if not self.app_info:
```

Comment:
Can this automatically call it if not already done? Will simplify tests and other stuff.